people/occupationsall.py

```
@@ -590,3 +604,8 @@
 # ---
 
 # ---
+
+if __name__ == "__main__":
+    # python3 core8/pwb.py people/occupationsall
+    for k in translationsOccupations.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)